docker-compose.yml

```diff
@@ -1,306 +1,389 @@
+    restart: unless-stopped
+    env_file:
+      - ./config/express-common.env
+    labels:
+      ca.mcgill.a11y.image.handler: enable
+
+  # For environment setup refer to https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/ocr
+  ocr-clouds-preprocessor:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-ocr-clouds:${REGISTRY_TAG}
+    restart: unless-stopped
+    env_file:
+      - ./config/apis-and-selection.env
+    labels:
+      ca.mcgill.a11y.image.preprocessor: 4
+      ca.mcgill.a11y.image.port: 5000
+
+  object-depth-calculator:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-object-depth-calculator:${REGISTRY_TAG}
+    restart: unless-stopped
+    labels:
+      ca.mcgill.a11y.image.preprocessor: 4
+      ca.mcgill.a11y.image.port: 5000
+
+  action-recognition:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-action-recognition:${REGISTRY_TAG}
+    restart: unless-stopped
+    labels:
+        ca.mcgill.a11y.image.preprocessor: 4
+        ca.mcgill.a11y.image.port: 5000
+    deploy:
+      resources:
+        reservations:
+          devices:
+          - driver: nvidia
+            device_ids: ['1']
+            capabilities: ["gpu", "utility", "compute"]
+
+  ocr-handler:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-handler-ocr:${REGISTRY_TAG}
+    restart: unless-stopped
+    labels:
+      ca.mcgill.a11y.image.handler: enable
+
+  osm-streets-handler:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-handler-osm-streets:${REGISTRY_TAG}
+    restart: unless-stopped
+    env_file:
+        - ./config/express-common.env
+    depends_on:
+        - supercollider
+        - espnet-tts
+    labels:
+        ca.mcgill.a11y.image.handler: enable
+    volumes:
+        - sc-store:/tmp/sc-store
+
+  svg-open-street-map-handler:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-handler-svg-open-street-map:${REGISTRY_TAG}
+    restart: unless-stopped
+    labels:
+        ca.mcgill.a11y.image.handler: enable
+        ca.mcgill.a11y.image.port: 5000
+
+  collage-detector-preprocessor:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-collage-detector:${REGISTRY_TAG}
+    restart: unless-stopped
+    labels:
+        ca.mcgill.a11y.image.preprocessor: 1
+        ca.mcgill.a11y.image.port: 5000
+
+  svg-action-recognition-handler:
+    profiles: [test, default]
+    image: ghcr.io/shared-reality-lab/image-handler-svg-action-recognition:${REGISTRY_TAG}
+    labels:
+        ca.mcgill.a11y.image.handler: enable
+
+# end - unicorn exclusive services        
 volumes:
-    sc-store:
-    user-logs:
+  sc-store:
+  user-logs:
+  website-logs:
+# https://docs.docker.com/compose/networking/
+networks:
+  traefik:
```

Comment:
I'm not sure about including traefik here, as this is sort of a specific thing we do...maybe include a note on _why_ it's here and how it can be removed/replaced? Or move to the prod/test yml files? Thoughts, @jeffbl?

Reply:
@JRegimbal I agree this is awkward, since it isn't really the IMAGE server stack itself, yet I do think we should have a working example, even if it does only work with one specific reverse proxy (traefik). Perhaps we should add comments to this indicating that traefik is only one reverse proxy, and this can be removed if the server is hosted on a "normal" webserver directly? Again, my preference is to have a fully working example, with (as you suggest) comments to explain potential alternatives. Doing a separate "vanilla" version is unlikely to be kept up-to-date, so may be even more confusing in the long run.